crates/app/tests/mod.rs

```diff
@@ -62,7 +66,10 @@ async fn app() {
         &foo,
         &tag,
         TagEntry::Unsigned(TreeEntry {
-            digest: Default::default(), // TODO: Set and require digest
+            digest: Algorithms::default()
+                .read(b"testing".as_slice())
```

Comment:
I don't think `.as_slice()` is necessary here? Byte string literals should automatically coerce to slices.

Reply:
It's required, otherwise: `the trait bound `&[u8; 7]: futures::AsyncRead` is not satisfied`